lib/hanami/action/mime.rb

```diff
@@ -165,6 +171,38 @@ def accept(*formats)
             end
           end
         end
+
+        # Restrict the payload type to the specified mime type symbols.
+        #
+        # @param formats[Array<Symbol>] one or more symbols representing mime type(s)
+        #
+        # @since x.x.x
+        #
+        # @example
+        #   require 'hanami/controller'
+        #
+        #   class Upload
+        #     include Hanami::Action
+        #     content_type :json
+        #
+        #     def call(params)
+        #       # ...
+        #     end
+        #   end
+        #
+        #   # When called with "text/html"        => 415
+        #   # When called with "application/json" => 200
+        def content_type(*formats)
+          mime_types = formats.map { |format| format_to_mime_type(format) }
+
+          before do
+            mime_type = @_env[HTTP_CONTENT_TYPE] || default_content_type || DEFAULT_CONTENT_TYPE
+
+            unless mime_types.find {|mt| ::Rack::Mime.match?(mime_type, mt) }
```

Comment:
Is this the same as:
```ruby
if mime_types.none? {|mt| ::Rack::Mime.match?(mime_type, mt) }
```
We're not using the resulting value, so I think a boolean reveals the intent better. Also, using `none?` instead of `any?` lets us use an `if` instead of `unless` which is :100: for me :)

I'm not 100% sure this is the same logic but I think it's equivalent?

Reply:
Thanks, will add a test and check can we replace it with `none` or so 👍